static/templates/admin/editStreamingPropertiesForm.html

```diff
@@ -0,0 +1,28 @@
+<iframe id="upload_file_frame" name="upload_file_frame" height="0" width="0" style="display: none;"></iframe>
+<form id="edit_streaming_properties" class="edit_form edit_streaming_properties_form">
+    <div class="form_field">
+        <label for="streaming_file"><span class="required">*</span> Streaming File:</label>
+        <input id="streaming_file" type="text" name="streaming_file" value="<%= metadata.streamingFile %>"/>
```

Comment:
Sharon's PR for adding a GET endpoint is probably we would want to populate the form, which would ensure that the values are up to date with what's in fedora. We could reconcile the two after they are both merged